docs/documentation/full-text/phrase.mdx

```diff
@@ -42,4 +42,5 @@ WHERE description @@@ '"plastic keyb"*';
 
 ## Special Characters
 
-The special characters `+` , `^`, `` ` ``, `:`, `{`, `}`, `"`, `[`, `]`, `(`, `)`, `~`, `!`, `\\`, `\*`, and `SPACE` must be escaped by a`\` inside the query term.
+The special characters `+` , `^`, `` ` ``, `:`, `{`, `}`, `"`, `[`, `]`, `(`, `)`, `<`, `>`, `~`, `!`, `\\`, `\*` must be
```

Comment:
I also posted the results of my experimentation with `\` at https://github.com/paradedb/paradedb/pull/1819#discussion_r1807111643, showing seemingly different behavior of it in phrase vs term search, so not sure what to make of it.

Also not sure if  `~` should be listed  in term search? At least with this example, escaping it turns out to be wrong:

(`whitespace` tokenizer)
```sql
=> CALL paradedb.drop_bm25( index_name => 'search_idx', schema_name => 'public');
CALL
=> CALL paradedb.create_bm25(
  index_name => 'search_idx',
  table_name => 'mock_items',
  schema_name => 'public',
  key_field => 'id',
  text_fields => paradedb.field('description')
|| paradedb.field('category', tokenizer=>paradedb.tokenizer('whitespace')),
  numeric_fields => paradedb.field('rating')
);
CALL
=> INSERT INTO mock_items (description, rating, category, in_stock, metadata, created_at, last_updated_date, latest_available_time)
VALUES ('Strange category item', 5, 'foot~wear', true, '{"color": "red", "size": 42}', NOW(), CURRENT_DATE, '10:00:00');
INSERT 0 1
=> SELECT description, rating, category                                                                                           FROM mock_items
WHERE id @@@ paradedb.parse('category:foot~wear') order by id desc limit 4;
      description      | rating | category
-----------------------+--------+-----------
 Strange category item |      5 | foot~wear
(1 row)

=> SELECT description, rating, category
FROM mock_items
WHERE id @@@ paradedb.parse('category:foot\~wear') order by id desc limit 4;
 description | rating | category
-------------+--------+----------
(0 rows)
```

So if you don't escape it, you get the result you want, if you escape it you get no matches.